docs/reporting/response-codes.asciidoc

```diff
@@ -0,0 +1,23 @@
+The Reporting APIs use HTTP response codes to indicate to give feedback. In automation,
+this helps external systems track the various possible job states:
+
+- **`200` (OK)**: As expected, Kibana returns `200` status in the response for
+  successful requests to queue or download reports.
++
+NOTE: Kibana will send a `200` response status for successfully queuing a Reporting job via
+the POST URL. This is true even if the job somehow fails later, since report
+generation happens asynchronously from queuing.
+
+- **`400` (Bad Request)**: When sending requests to the POST URL, if you don't use
+  `POST` as the HTTP method, or if your request is missing the `kbn-version` header,
+  Kibana will return a code `400` status response for the request.
+
+- **`503` (Service Unavailable)**: When using the `path` to request the download, you will get
+  a `503` status response if report generation hasn't completed yet. In that case,
+  retry after the number of seconds in the `Retry-After` header in the download API
```

Comment:
This sentence is confusing:

In that case, retry after the number of seconds in the `Retry-After` header in the download API response until the report is complete.

Reply:
I see what you mean. How about:

> When using the `path` to request the download, you will get a `503` status response if report generation hasn't completed yet. A `503` response comes with a `Retry-After` header. Before trying again to download the report, you can have your script wait the suggested number of seconds found in the `Retry-After` header until the report is complete.